### ISSUE TITLE
Apposition, acl and clauses

### ISSUE THREAD

**Issue by GPPassos** [UniversalDependencies/docs#523]
In the [appos page](http://universaldependencies.org/u/dep/appos.html), it is stated that

> While appos is normally between two nominals, there are a few cases where there is a relation with a clause, such as when describing facts or events for which appos still feels appropriate:

An example is given:
> This problem , that people could lower their tax rates by choosing to become corporations , might become acute .
> appos(problem-2, lower-7)

However, there is a very similar construction in the [acl page](http://universaldependencies.org/u/dep/acl.html):
> Some languages allow finite clausal complements for nouns with a subset of nouns like fact or report. These look roughly like relative clauses, but do not have any omitted role in the dependent clause. This is the class of “content clauses” in Huddleston and Pullum 2002). These are also analyzed as acl.

> the fact that nobody cares
> acl(fact, cares)

Is there any significant difference between the two examples?

**Comment by jnivre** [comment]
I would say that the first is a mistake. (I don't know who wrote it.) But the overview table of relations makes very clear that an "appos" should itself be a noun phrase, not a clause. Hence, "acl" is the right relation for this cases.

**Comment by dan-zeman** [comment]
> While appos is normally between two nominals, there are a few cases where there is a relation with a clause, such as when describing facts or events for which appos still feels appropriate

> I would say that the first is a mistake. (I don't know who wrote it.)

Git blame says it was @manning in https://github.com/UniversalDependencies/docs/commit/eed6397d7a1f5149f2a1970bdb402aa35e8c2442 :-)

**Comment by jnivre** [comment]
I still think it is a mistake. :)

**Comment by GPPassos** [comment]
Should this be closed? I also agree that this is a mistake and should be removed, but I don't know if and how this could be done (as I don't know which kinds of changes are allowed to the documentation, for instance due to versioning, etc)

**Comment by martinpopel** [comment]
> Should this be closed?

No, I think issues like this one should be closed only after the documentation is fixed. Ideally, it should be closed by merging a PR which fixes the issue.

> I don't know if and how this could be done

Each documentation page has an "edit page" button which allows to propose a change. I just used it and created a PR which deletes the mistaken example. (Anyone can open a new PR with an added sentence explaining that `acl` is suitable for such cases.)

> which kinds of changes are allowed to the documentation

Fixing error and inconsistencies and explaining the intended guidelines is allowed anytime. Major changes (esp. backward incompatible) are allowed only with major version bump (e.g. UD1 -> UD2).
